src/rules/props-order.ts

```diff
@@ -97,32 +141,26 @@ const sortAttributes = (unsorted: (JSXAttribute | JSXSpreadAttribute)[]) => {
       end = i + 1;
       if (start < end) {
         const sectionToSort = unsorted.slice(start, end) as JSXAttribute[];
-        const sectionSorted = sectionToSort.sort((a, b) => compare(a, b));
+        const sectionSorted = sectionToSort.sort((a, b) => compare(a, b, config));
         sorted = sorted.concat(sectionSorted);
       }
     }
   }
   return sorted;
 };
 
-const compare = (a: JSXAttribute, b: JSXAttribute) => {
-  const aPriority = getPriorityIndex(a.name.name.toString());
-  const bPriority = getPriorityIndex(b.name.name.toString());
+const compare = (a: JSXAttribute, b: JSXAttribute, config: Config) => {
+  const aPriority = getPriority(a.name.name.toString(), config);
+  const bPriority = getPriority(b.name.name.toString(), config);
 
   if (aPriority !== bPriority) {
     return aPriority - bPriority;
   }
 
   // Same Priority. Then compare it.
-  const priority = aPriority;
-  const order = priority <= priorityGroupsLength ? "predefined" : "alphabetical order";
-  switch (order) {
-    case "predefined": {
-      const aIndex = getIndexInPriority(a.name.name.toString(), aPriority);
-      const bIndex = getIndexInPriority(b.name.name.toString(), bPriority);
-
-      return aIndex - bIndex;
-    }
+  const ordering = "alphabetical order";
+
+  switch (ordering) {
     case "alphabetical order":
       return a.name.name < b.name.name ? -1 : 1;
```

Comment:
The switch statement is not necessary. But I decided to keep it because it clarifies `a.name.name < b.name.name` is a alphabetical compare,  and it remains easy to extend.